mne/io/base.py

```diff
@@ -1661,6 +1662,11 @@ def save(self, fname, picks=None, tmin=0, tmax=None, buffer_size_sec=None,
             .. note:: Due to FIFF file limitations, the maximum split
                       size is 2GB.
 
+        split_naming : string ('neuromag' | 'bids')
```

Comment:
Sorry I missed this one, curly braces not `()` like `split_naming : {'neuromag', 'bids'}` https://numpydoc.readthedocs.io/en/latest/format.html